### DISCUSSION TITLE
能不能接入fastgpt?

### DISCUSSION THREAD

**Discussion by rainberg** [zhayujie/chatgpt-on-wechat#1129] [Ideas]
https://github.com/c121914yu/FastGPT

能不能开发下，可以接入fastGPT

**Comment by zlinoliver** [comment]
同问，[fastgpt](https://github.com/labring/FastGPT) 的知识库内容管理 比 [LinkAI](https://chat.link-ai.tech/console) 更好, 期待大佬接入 @zhayujie
